.github/workflows/raise-pr-for-constants.yml

```
@@ -0,0 +1,72 @@
+
+      - name: Checkout Destination Repo
+        run: |
+          DEST_REPO="rudderlabs/rudder-sdk-js"
+          git clone https://github.com/${DEST_REPO}.git dest-repo
+          cd dest-repo
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "noreply@github.com"
+          git config --global url."https://${{ secrets.GITHUB_TOKEN }}@github.com/".insteadOf "https://github.com/"
+          git checkout -b ${{ env.BRANCH_NAME }}
+          git pull origin ${{ env.BRANCH_NAME }} || echo "No updates to pull"
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)